currency-v1/apiproxy/resources/jsc/ConfigureTargetRequest.js

```diff
@@ -1 +1,3 @@
 context.setVariable('target.copy.pathsuffix', false);
+
+context.setVariable('bella', 'cool');
```

Comment:
this is a pointless of code - but its the end of the sprint so i will merge it anyway.